build-gradle-plugin/gradle/libs.versions.toml

```diff
@@ -0,0 +1,18 @@
+[versions]
+androidGradlePlugin = "7.3.1"
+kotlin = "1.7.10"
```

Comment:
I wanted too, but unfortunately it's not possible. Because gradle plugin expects `2.4.1` version of the `kotlin-dsl` and dsl expects kotlin `1.7.10` :smirk:  Otherwise it shows warning, or even fails on build